include/zephyr/bluetooth/att.h

```diff
@@ -91,6 +91,12 @@ size_t bt_eatt_count(struct bt_conn *conn);
 
 #endif /* CONFIG_BT_EATT */
 
+enum bt_att_chan_option {
```

Comment:
Is “option” a term that the spec uses? If not, I think “preference” (shortened to “pref”) would be more intuitive.

Reply:
Agree "preference" is more intuitive for the current use of this field.

Reply:
Doesn't `preference` imply that the host may actually use e.g. an unenhanced ATT bearer, even if `BT_ATT_CHAN_ENHANCED` is the only bit set? 

`option` is the term we use for e.g. the advertising options in `bluetooth.h`, and I think the term `option` better describe what this value represent.

Reply:
@jhedberg @Thalley do you agree to `prefs` for the naming ?

Reply:
If we want the host to _control_ which type of bearer, then `prefs` is a poor choice. As mentioned in https://github.com/zephyrproject-rtos/zephyr/pull/45332#discussion_r869963284, `prefs` implies that it may be overridden by the stack, which is not only incorrect (as of the current implementation), but also does not the only issue that we have that requires this feature. 

I vote for `options` to be consistent with the options used in `bluetooth.h`.

Reply:
I'd be ok with `options`, what do you think @jhedberg ?